isolate/02-ooping-1/exercises/2-read-from.js

```diff
@@ -2,8 +2,12 @@
 
 const obj = {
   word: '',
-  getWord: function () { },
-  concat: function (secondHalf) { },
+  getWord: function () {
+    return this.word = 'the word is ';
```

Comment:
the goal of getWord is to combine a little explanation with the saved word (`test1`, `test4`, `test7`), how would you do this with a regular function and `word` as an argument?  

```js
const getWord = (word) => {
    // ?
};

getWord('tree'); // -> 'the word is tree'
```

then you can refactor to a method.